helix-rest/src/main/java/org/apache/helix/rest/clusterMaintenanceService/MaintenanceManagementService.java

```diff
@@ -159,13 +159,10 @@ private MaintenanceManagementService(ZKHelixDataAccessor dataAccessor,
     _customRestClient = customRestClient;
     _skipZKRead = skipZKRead;
     _nonBlockingHealthChecks =
-        continueOnFailure ? Collections.singleton(ALL_HEALTH_CHECK_NONBLOCK)
-            : Collections.emptySet();
+        nonBlockingHealthChecks == null ? Collections.emptySet() : nonBlockingHealthChecks;
     _skipHealthCheckCategories =
-        skipHealthCheckCategories == null ? Collections.emptySet()
-            : skipHealthCheckCategories;
-    _stoppableHealthCheckList =
-       stoppableHealthCheckList == null ? Collections.emptyList()
+        skipHealthCheckCategories == null ? Collections.emptySet() : skipHealthCheckCategories;
```

Comment:
Please consider to have homogeneous input. For example, we have flack list for skip customized check and whit list for helix health check. It is a bit confusing.

Reply:
change it from `stoppableCheckList` to be `skipStoppableCheckList`. And by default, no checks will be skipped.